src/codelabs/async-await/index.md

```diff
@@ -312,32 +313,49 @@ main() async {
 
 ## Practice using async and await
 
-
-
 The following example is a failing unit test that contains partially completed
 code snippets. Your task is to complete the example so that the test passes.
 Verify your code by clicking the "Run" button.
 
 Add the following:
 
-* Part 1: Add logic to the `reportUserRole` function so that it returns a Future
-  that produces the String result ‘User role: <user role>!'. The <user role> can
-  be obtained by calling the provided async function `getRole` which returns a
-  `String` describing the user role. Example return value from
-  `reportUserRole()`: `'User role: tester'`.
-* Part 2: Write an `async` function `reportLogins` that returns the string
-  'Total number of logins: <# of logins>'. You can obtain the number of logins
-  by calling the provided asynchronous function `getLoginAmount`, which returns
-  an `int` representing the number of times that the user has logged in. Example
-  return value from `reportLogins(): 'Total number of logins: 57'`.
+#### Part 1
+Add logic to the `reportUserRole` function:
 
+1. `reportUserRole` returns a Future that produces the String result `'User role: <user role>'`.
+1. Obtain the user role by calling the provided `async` function `getRole`
+1. `getRole` returns a `String` describing the user's role.
+1. Example return value from `reportUserRole`: `'User role: tester'`
 
-## Handling errors
+#### Part 2
+Implement an `async` function `reportLogins`:
 
+1. `reportLogins` returns the string `'Total number of logins: <# of logins>'`.
+1. Obtain the number of logins by calling the provided `async` function `getLoginAmount`
+1. `getLoginAmount` returns an `int` representing the number of times that the user has logged in.
+1. Example return value from `reportLogins: 'Total number of logins: 57'`
 
+<iframe frameborder="no" height="525" src="https://dartpad2-ux.firebaseapp.com/experimental/embed-new?id=f751b692502c4ee43d932f745860b056" width="100%"></iframe>
```

Comment:
Use production links for Dartpad2 instead of the UX experimental site. 
https://dartpad.dartlang.org/experimental/embed-new.html?id=f751b692502c4ee43d932f745860b056